cdap-distributions/src/etc/cdap/conf.dist/cdap-site.xml.example

```diff
@@ -17,119 +17,152 @@
+    <value>0.0.0.0</value>
+    <description>
+      Transaction service inet address
+    </description>
   </property>
 
-  <!-- 
-    Kafka Configuration
-  -->
+
+  <!-- Kafka Server Configuration -->
+
+  <property>
+    <name>kafka.default.replication.factor</name>
+    <value>1</value>
+    <description>
+      Kafka replication factor; used to replicate Kafka messages across
+      multiple machines to prevent data loss in the event of a hardware
+      failure. The recommended setting is to run at least two Kafka servers.
+      If you are running two Kafka servers, set this value to 2; otherwise,
+      set it to the number of Kafka servers.
+    </description>
+  </property>
+  
   <property>
     <name>kafka.log.dir</name>
-    <value>/data/cdap/kafka-logs</value>
-    <description>Directory to store Kafka logs</description>
+    <value>/tmp/kafka-logs</value>
+    <description>
+      Kafka log storage directory
+    </description>
   </property>
 
-  <!-- Substitute with a list of all machines which will run the Kafka component -->
   <property>
     <name>kafka.seed.brokers</name>
-    <value>FQDN1:9092,FQDN2:9092</value>
-    <description>List of Kafka brokers (comma separated)</description>
+    <value>127.0.0.1:9092</value>
+    <description>
+      List of Kafka brokers (comma separated)
+    </description>
   </property>
 
-  <!-- Must be <= the number of kafka.seed.brokers configured above.  For HA this should be at least 2. -->
-  <property>
-    <name>kafka.default.replication.factor</name>
-    <value>1</value>
-    <description>Kafka replication factor</description>
-  </property>
 
-  <!--
-    Watchdog Configuration
-  -->
-  <!-- Substitute the IP to which metrics-query service should bind to and listen on -->
+  <!-- Metrics Configuration -->
+  
   <property>
     <name>metrics.query.bind.address</name>
-    <value>LOCAL-WATCHDOG-IP</value>
-    <description>Specifies the inet address on which the metrics-query service will listen</description>
+    <value>0.0.0.0</value>
+    <description>
+      Metrics query server host address
+    </description>
   </property>
 
-  <!--
-    Web-App Configuration
-  -->
+
+  <!-- UI Configuration -->
+  
   <property>
     <name>dashboard.bind.port</name>
     <value>9999</value>
-    <description>Specifies the port on which dashboard listens</description>
+    <description>
+      CDAP UI bind port
+    </description>
   </property>
 
-  <!-- Substitute the IP of the Router service to which the UI should connect -->
   <property>
     <name>router.server.address</name>
-    <value>ROUTER-HOST-IP</value>
-    <description>Specifies the destination IP where Router service is running</description>
-  </property>
-
-  <property>
-    <name>router.server.port</name>
-    <value>10000</value>
-    <description>Specifies the destination Port where Router service is listening</description>
+    <value>127.0.0.1</value>
```

Comment:
While the default ``127.0.0.1`` should work most of the time, i think for the same reason as kafka we should force the user to replace a token (``ROUTER-HOST-IP`` or similar) here.  I think it's ok to have the bind addresses set to ``0.0.0.0``, but for these 2 settings that govern remote endpoints, we cannot assume anything about the user's cluster in a template.